modules/createTransitionManager.js

```diff
@@ -37,7 +37,7 @@ const createTransitionManager = () => {
         }
       } else {
         // Return false from a transition hook to cancel the transition.
-        callback(result !== false)
+        callback(false)
```

Comment:
Won't this result in always blocking the transition, even if `confirmLeave` from RFR (which I think is `prompt`) returns `true`? I would have thought

```javascript
callback(Boolean(result));
```
would be what is needed.

Reply:
I can implement that if @DavOnGit @hedgepigdaniel  agree?

Reply:
Update the PR and if possible, make a test for this case / extend an existing one

Reply:
I'll try to test it next days, ty! @hedgepigdaniel @ScriptedAlchemy

Reply:
Thanks